en/_index.md

```diff
@@ -31,20 +31,20 @@ The corresponding relationship between TiDB Operator and TiDB versions is as fol
 <ColumnTitle>Quick Start</ColumnTitle>
```

Comment:
2 suggestions:
1. Put Architecture under `About TiDB Operator`
2. Remove [Use tkctl](use-tkctl.md) from `Reference` because this tool is no longer maintained.
<img width="712" alt="image" src="https://user-images.githubusercontent.com/59654584/155664865-019db63f-3ed6-4223-86b4-9ab7327d4401.png">